application/src/main/java/org/togetherjava/tjbot/features/basic/SlashCommandEducator.java

```diff
@@ -19,6 +19,7 @@
  * then educates the user about using slash commands, such as {@code /foo} instead.
  */
 public final class SlashCommandEducator extends MessageReceiverAdapter {
+    private static final int MAX_CONTENT_LENGTH = 30;
```

Comment:
this is supposed to denote the max length for a possible command. so a better name would be `MAX_COMMAND_LENGTH` ( @Taz03 )